yatdlm/todo/models.py

```diff
@@ -280,6 +305,26 @@ def as_dict(self, dates_format="d/m/Y"):
 
         return resp
 
+    def update(self, data: dict, logged_user: User) -> None:
+        if logged_user != self.owner:
+            raise NotOwner()
+
+        priority = data.get("priority")
+        title = data.get("title")
+        description = data.get("description")
+
+        if "categories" in data:
+            categories = [int(category_id) for category_id in data["categories"]]
+            self.set_categories(categories)
```

Comment:
Personal preference would be to `self.set_categories(int(category_id) for ...)`.